xdsl/dialects/func.py

```diff
@@ -95,6 +96,17 @@ def from_region(
         op = FuncOp.build(attributes=attributes, regions=[region])
         return op
 
+    @staticmethod
+    def implicit_builder(
+        name: str,
+        input_types: Sequence[Attribute],
+        return_types: Sequence[Attribute],
+    ):
```

Comment:
Can you add a return type here, so it is clearer how to use it?
Or also just some simple documentation, like "create the operation using an implicit builder inside a `with` statement."